tests/test_entropy.py

```diff
@@ -25,17 +27,14 @@ def test_shannonEntropyInt(self):
         self.assertEqual(round(ent.shannon_entropy(TIME_SERIES), 5), SHANNON_ENTROPY)
 
     def test_sampleEntropy(self):
-        ts = TS_SAMPLE_ENTROPY
+        ts = np.array(TS_SAMPLE_ENTROPY)
         std_ts = np.std(ts)
-        sample_entropy = ent.sample_entropy(ts, 4, 0.2 * std_ts)
-        np.testing.assert_array_equal(np.around(sample_entropy, 8),
-                                      [2.21187685, 2.10787948, 2.36712361, 1.79175947])
+        sample_entropy = ent.sample_entropy(ts, 3, 0.2 * std_ts)
+        self.assertEqual(np.around(sample_entropy, 8), 1.79175947)
 
-    def test_multiScaleEntropy(self):
-        ts = TS_SAMPLE_ENTROPY
-        std_ts = np.std(ts)
-        multi_scale_entropy = ent.multiscale_entropy(ts, 4, 0.2 * std_ts)
-        np.testing.assert_array_equal(np.round(multi_scale_entropy, 4), np.array([2.2119, 2.6221, 1.7473, 1.9902]))
+    def test_multiScaleEntropy(self):  
+        multi_scale_entropy = ent.multiscale_entropy(RANDOM_TIME_SERIES, 3, maxscale = 4 )
```

Comment:
Formatting: redundant space

Reply:
I have changed the file (but I am not sure which redundant space you are referring to...)